mobile/src/modules/auth/screens/SendScreen.tsx

```diff
@@ -12,48 +13,72 @@ import { core } from '../../../core/translations';
 import { AuthReduxState } from '../store/reducer';
 
 interface IProps extends InjectedReduxProps {
-  auth: AuthReduxState,
+  auth: AuthReduxState;
 }
+
+interface SendBurstFormState {
+  recipientAddress: string;
+  amountNQT: string;
+  feeNQT: string;
+  showMessage: boolean;
+  advanced: boolean;
+  message: string;
+  encrypt: boolean;
+  deadline: string;
+  pin: string;
+}
+
 type Props = IProps & NavigationInjectedProps;
 
 class Send extends React.PureComponent<Props> {
   static navigationOptions = {
     headerTitle: <HeaderTitle>{i18n.t(core.screens.send.title)}</HeaderTitle>
   };
 
-  state = {
-    amountNQT: 0,
-    feeNQT: 0
+  state: SendBurstFormState = {
+    recipientAddress: '',
+    amountNQT: '0',
+    feeNQT: '0',
+    showMessage: false,
+    advanced: false,
+    message: '',
+    encrypt: true,
+    deadline: '24',
+    pin: ''
   };
 
+  private handleSubmit = () => {
+    console.log(this.state.recipientAddress, this.state.amountNQT, this.state.feeNQT);
+  }
+  private handleInputChange (key: keyof SendBurstFormState): ((text: string) => void) {
+    return (text) => this.setState({ [key]: text });
+  }
+  private handleBurstInputChange (): ((text: string) => void) {
+    return (text) => this.setState({ recipientAddress: text });
+  }
+
   render () {
     return (
```

Comment:
what if `const {recipientAddress, amountNQT, feeNQT} = this.state;`? Destructuring assignment keeps code simpler and shorter